### ISSUE TITLE
Redundant __init__.py files in core.xhtml

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#3297]
|<img alt="tyrion-mx's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| tyrion-mx reported|
|-|-|
|Trac ID|trac#3297|
|Type|defect|
|Created|2008-06-17 16:25:00Z|

There are some unneeded ```__init__.py``` files in the directory tree located at core.xhtml#auto12.
In the attached patch I also replaced all the ```core_special.py``` occurrencies with ```myapp_plugins.py``` as exarkun suggested.

Attachments:

* [cred.xhtml.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/6b9/6b9c18771e13d6e05a9af3e3b160f37e6dfb9958/9cc3969db113cb178598a4ec5ac934db4e65c138.patch) (1194 bytes) - added by tyrion-mx on 2008-06-17 16:31:52Z - 

<details><summary>Searchable metadata</summary>

```
trac-id__3297 3297
type__defect defect
reporter__tyrion_mx tyrion-mx
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords____init___py_core_xhtml __init__.py core.xhtml
time__1213719900000000 1213719900000000
changetime__1213804084000000 1213804084000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="tyrion-mx's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|tyrion-mx commented|
|-|-|

I also edited the last example's code moving the "zope.interface" import before the twisted block of imports, as I've seen on all other examples in the doc.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

(In [23952]) Apply cred.xhtml.patch

Author: tyrion-mx
Reviewer: exarkun
Fixes: [#3297](https://github.com/twisted/twisted/issues/3297)

Remove the detrimental `__init__.py` files from the example application
filesystem structure so that the plugin it defines will actually be
found and used. Also rename the dropin file to reflect the preference for
a single dropin file per project, rather than per plugin or plugin type.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
